File: app/src/main/java/com/games/memorygame/util/DateTimeUtil.java
package com.games.memorygame.util;


import android.content.Context;

import com.games.memorygame.R;

import java.util.concurrent.TimeUnit;

public class DateTimeUtil {

    public static String fromMilliSecondsToString(Context context, long millis) {
        return context.getString(R.string.time_format,
                TimeUnit.MILLISECONDS.toMinutes(millis),
                TimeUnit.MILLISECONDS.toSeconds(millis) -
                        TimeUnit.MINUTES.toSeconds(TimeUnit.MILLISECONDS.toMinutes(millis)));
    }
}

File: app/src/main/java/com/games/memorygame/util/DateTimeUtil.kt
package com.games.memorygame.util


import android.content.Context

import com.games.memorygame.R

import java.util.concurrent.TimeUnit

object DateTimeUtil {

    fun fromMilliSecondsToString(context: Context, millis: Long): String {
        return context.getString(R.string.time_format,
                TimeUnit.MILLISECONDS.toMinutes(millis),
                TimeUnit.MILLISECONDS.toSeconds(millis) - TimeUnit.MINUTES.toSeconds(TimeUnit.MILLISECONDS.toMinutes(millis)))
    }
}
